bind_slurm.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Specify system-specific bindings to use here.
+
+CONFIG=nesh.conf
+
+
+
+# Load bindings from config.
+
+source ${CONFIG}
+
+echo Will use: ${CONFIG}
+
+# Check if singularity is available.
+
```

Comment:
Delete additional whitespace? (For me, syntax highlighting makes things like this easy to read without additional space.)
```suggestion
```